app/profile/profile.html

```diff
@@ -0,0 +1,217 @@
+            <p>Started to work: {{userStartedToWork}}</p>
+            <p>Birthday: {{userBirthday}}</p>
+
+            <button type="button" class="btn btn-default btn-sm">Edit</button>
+        </div>
+
+        <div class="col-md-6">
+
+            <div class="row">
+                <div class="col-xs-3">
+                    <h3><b>{{userKudos}}</b></h3>
+                    <h4>Corns</h4>
+                </div>
+                <div class="col-xs-3">
+                    <h3><b>{{userReceivedKudos}}</b></h3>
+                    <h4>Received Corns</h4>
+                </div>
+                <div class="col-xs-3">
+                    <h3><b>{{userKudos}}</b></h3>
+                    <h4>Challenges</h4>
+                </div>
+            </div>
+
+            <h3>Corns</h3>
+
+            <table class="table">
+                <thead>
+                <tr>
+                    <th>Colleague</th>
+                    <th>Acorns</th>
+                    <th>Message</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr ng-repeat="incomingKudos in incomingKudosCollection">
+                    <td>{{incomingKudos.senderEmail}}</td>
+                    <td>{{incomingKudos.amount}}</td>
+                    <td>{{incomingKudos.message}}</td>
+                </tr>
+                </tbody>
+            </table>
+            <button type="button" class="btn btn-default btn-sm">Show more</button>
+
+            <h3>Endorsements</h3>
+            <button type="button" class="btn btn-default btn-sm">Show more</button>
+
+            <h3>Challenges</h3>
+            <table class="table">
+                <thead>
+                <tr>
+                    <th>Colleague</th>
+                    <th>Chalenge</th>
+                    <th>Status</th>
+                    <th>Chalenge Date</th>
+                    <th>Description</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr>
+                    <td>Lorem</td>
+                    <td>Ipsum</td>
+                    <td>Dolores</td>
+                    <td>2020-20-20</td>
+                    <td>Sit</td>
+                </tr>
+                </tbody>
+            </table>
+            <button type="button" class="btn btn-default btn-sm">Show more</button>
+        </div>
+
+        <div class="col-md-3">
+            <div class="btn-group-vertical" role="group" aria-label="...">
+                <button ng-click="showSendKudosModal()" data-toggle="modal" data-target="#sendKudosModal" type="button" class="btn btn-default">Give Acorn</button>
+                <button type="button" class="btn btn-default">Endorse</button>
+                <button type="button" class="btn btn-default dropdown-toggle">Challenge</button>
+            </div>
+
+            <h4>Today's birthday</h4>
+            <p>Lorem</p>
+
+            <h4>Challenge winner</h4>
+            <p><u>Foosball</u></p>
```

Comment:
The HTML 5 specification reminds developers that other elements are almost always more appropriate than <u>.